android/build_all.sh

```diff
@@ -17,4 +17,19 @@ cd android/
 ./buildAndroidBOINC-CI.sh --cache_dir "$ANDROID_TC" --build_dir "$BUILD_DIR" --arch x86
 ./buildAndroidBOINC-CI.sh --cache_dir "$ANDROID_TC" --build_dir "$BUILD_DIR" --arch x86_64
 
+if [ ! -d ../3rdParty/buildCache/android-tc/arm ]; then
+    echo You run this script from diffrent folder: $PWD
+    echo Please run it from boinc/android
+    exit 1
+fi
+
+list_libs="libcrypto.a libssl.a libcurl.a"
+
+for i in $list_libs; do
+    if [ $(readelf -A $(find ../3rdParty/buildCache/android-tc/arm  -name "$i") | grep -i neon | head -c1 | wc -c) -ne 0 ]; then
```

Comment:
This path could be undefined if `--cache_dir` is used to target any other folder

Reply:
I can add another check.
If the ANDROID_TC is define, you it. otherwise you this.
That ok?

Reply:
Fix it.

Reply:
No, you should ${PREFIX} always because this could be any location. If this path is set by default this doesn't mean that it will never change in future

Reply:
${PREFIX} is not define here. ${PREFIX} is define by this $ANDROID_TC in buildAndroidBOINC-CI.sh
I think if I use here a ${PREFIX} it always be undefined. 
Because  buildAndroidBOINC-CI.sh is doing this line: `export ANDROID_TC=$PREFIX`
And $ANDROID_TC here undefined. 
So it better like I am doing.